controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3332,35 +3333,17 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
     @Message(id = 398, value = "An attempt was made to rename the resource found at %s to %s. However, '%s' is one of the resource types defined to be ordered on the parent resource %s")
     OperationFailedRuntimeException orderedChildTypeRenamed(PathAddress read, PathAddress transformed, String type, Set<String> parentOrderedChildren);
 
-    @Message(id = 399, value = "Model references of type '%s' are missing: %s")
-    OperationFailedException missingReferences(String type, Set<String> missing);
```

Comment:
In 1.0.0.Final (i.e. WF 9) the ids end at 395, so we should be good.
https://github.com/wildfly/wildfly-core/blob/1.0.0.Final/controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

(Not that I actually checked beforehand :-) )